designsafe/apps/audit/views.py

```diff
@@ -0,0 +1,121 @@
+"""Views for the audit app."""
+from django.shortcuts import render
+from django.http import JsonResponse
+from django.db import connections
+from django.contrib.auth.decorators import login_required
+
+@login_required
+def audit_trail(request):
+    """
+    Audit trail view - renders React component for audit functionality
+    """
+    context = {'title': 'Audit Trail'}
+    return render(request, 'designsafe/apps/audit/audit_trail.html', context)
+
+def get_portal_audit_search(request, username):
+    """
+    Fetches audit records for given username from portal audit database
+    """
+    try:
+        audit_db = connections['audit']
+        cursor = audit_db.cursor()
+
+        #for "joyce_cywu" search(many results) -- TEST -- won't work for other usernames
```

Comment:
I'm glad you thought ahead about how to test this stuff, but we can't commit the PR with this test data in there. Do you think you can create some unit tests for your new feature? Can any of this data be moved to testing steps or instructions for the developers to do locally?